modules/account-lib/test/unit/coin/eth2/keyPair.ts

```diff
@@ -128,39 +128,36 @@ describe('Eth2 Key Pair', () => {
   });
 
   describe('Signatures', () => {
-    it('should accept 2-of-2 aggregated signature', () => {
+    it('should accept 2-of-2 aggregated signature', async () => {
       const msg = Buffer.from('test message');
       const keyPair1 = new KeyPair();
       const pub1 = Uint8Array.from(Buffer.from(keyPair1.getKeys().pub.slice(2), 'hex'));
-      const sec1 = Uint8Array.from(Buffer.from(keyPair1.getKeys().prv!.slice(2), 'hex'));
-      const sig1 = BLS.sign(sec1, msg);
+      const sig1 = await keyPair1.sign(msg);
 
       const keyPair2 = new KeyPair();
       const pub2 = Uint8Array.from(Buffer.from(keyPair2.getKeys().pub.slice(2), 'hex'));
-      const sec2 = Uint8Array.from(Buffer.from(keyPair2.getKeys().prv!.slice(2), 'hex'));
-      const sig2 = BLS.sign(sec2, msg);
+      const sig2 = keyPair2.sign(msg);
```

Comment:
do we need to await sig2 as well?

Reply:
ah good catch actually we don't need to await either